pkg/registry/core/service/ipallocator/allocator_test.go

```diff
@@ -166,18 +175,66 @@ func TestAllocateSmall(t *testing.T) {
 }
 
 func TestRangeSize(t *testing.T) {
-	testCases := map[string]int64{
-		"192.168.1.0/24": 256,
-		"192.168.1.0/32": 1,
-		"192.168.1.0/31": 2,
+	testCases := []struct {
+		name   string
+		cidr   string
+		addrs  int64
+		expect bool
+	}{
+		{
+			name:   "success IPv4 /24 cidr",
+			cidr:   "192.168.1.0/24",
+			addrs:  256,
+			expect: true,
+		},
+		{
+			name:   "failure IPv4 /24 cidr",
+			cidr:   "192.168.1.0/24",
+			addrs:  512,
```

Comment:
Comparing this test case ("failure IPv4 /24 cidr") to the previous, I'm not sure this test case adds much value. In the previous test case, you check that RangeSize returns the expected 256. Now in this test case, you pass exactly the same CIDR as before, and compare the result (which you know is going to be 256) with 512, which you know is going to fail (i.e no additional test coverage).

Reply:
I have removed the v4 an v6 test cases that are redundant.